### PR TITLE
WL-3725 : Expand templates if property is set

### PR DESCRIPTION
**Pull request by nicholaswilson100**

### CODE REVIEW

File: site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java [hunk 1]
```diff
@@ -1425,7 +1425,7 @@ private String buildContextForTemplate(String preIndex, int index, VelocityPortl
 		String alert=(String)state.getAttribute(STATE_MESSAGE);
 		context.put("alertMessage", state.getAttribute(STATE_MESSAGE));
 		context.put("siteTextEdit", new SiteTextEditUtil());
-		
+		context.put("isExpandTemplates", ServerConfigurationService.getBoolean("add.site.expand.templates", false));
```
Comment (buckett) [inline site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java]: I think generally config properties should be prefixed by the tool name when they are specific to a tool. So maybe `sitemanage.add.exand.template`

**Comment by nicholaswilson100** [comment]
ok

**Comment by nicholaswilson100** [comment]
thanks

**Comment by nicholaswilson100** [comment]
svn property change committed

**Comment by buckett** [comment]
Do you want to try to re-base this and squash it into one commit?

**Comment by nicholaswilson100** [comment]
yup

**Comment by nicholaswilson100** [comment]
ok

**Comment by buckett** [comment]
Magic. :cake:

**Pull request merged**